jobsdb/jobsdb.go

```diff
@@ -2611,6 +2611,135 @@ func (jd *HandleT) GetUnprocessed(customValFilters []string, count int, paramete
 	return outJobs
 }
 
+/*
+DeleteJobStatus deletes the status of a batch of jobs
```

Comment:
DeleteJobStatus deletes the `latest` status of a batch of jobs

Reply:
done